packages/backend-core/src/api/collection/OrganizationApi.ts

```diff
@@ -9,6 +9,11 @@ type OrganizationMetadataParams = {
   privateMetadata?: Record<string, unknown>;
 };
 
+type GetOrganizationListParams = {
```

Comment:
🙃 
At a next PR let's see if we can add an interface enhancer for Pagination params and not repeat them across.